util/test/prediff-for-incremental-warning

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+#
+# This script is a system-wide prediff for use when stack tracing is enabled.
+# If the test doesn't present a stack trace in the expected output, this script
+# will remove the stack trace (if present) in the generated output.
+#
+# Can also be used with the environment variable (on bash)
+#
+#  export CHPL_SYSTEM_PREDIFF=$CHPL_HOME/util/test/prediff-for-stacktrace
```

Comment:
I'll start testing. In the meantime, it would be good to update some of these comments. Someone reading this file in the future may be confused by the mentions of stack tracing.